README.md

```diff
@@ -51,6 +51,18 @@ To update these docs, you'll need to:
 3. Run `HOME='~' helm docs --type markdown --generate-headers` to generate the markdown docs files, replacing any prior markdown files.  **Note:** Make sure you run the version of helm you want to build the docs for (for example, by checking out the correct tag and building it)
 4. Commit the changes and create a PR to update the website.
 
+> [!TIP]
+> If you are on macOS and want to avoid unnecessary changes to the config location, you can use this Docker command from the `content/en/docs/helm/` folder
+> ```
+> docker run --rm -v $(pwd):/output \
+>  --entrypoint /bin/sh \
+>  alpine:latest -c "
+>    apk add --no-cache curl bash && \
+>    curl -fsSL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash && \
+>    export HOME='~' && \
+>    helm docs --type markdown --generate-headers --dir /output
+>  "
+> ```
```

Comment:
I'm getting this:
```
Executing busybox-1.37.0-r18.trigger
OK: 17 MiB in 29 packages
In order to verify checksum, openssl must first be installed.
Please install openssl or set VERIFY_CHECKSUM=false in your environment.
Failed to install helm
	For support, go to https://github.com/helm/helm.
% echo $?
1
```

Reply:
That's weird. 🤔 

I just tested locally with `ubuntu` instead of `alphine` and it's working well.

```sh
Processing triggers for libc-bin (2.39-0ubuntu8.4) ...
Processing triggers for ca-certificates (20240203) ...
Updating certificates in /etc/ssl/certs...
0 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d...
done.
[WARNING] Could not find git. It is required for plugin installation.
Downloading https://get.helm.sh/helm-v3.18.3-linux-arm64.tar.gz
Verifying checksum... Done.
Preparing to install helm into /usr/local/bin
helm installed into /usr/local/bin/helm
```

And output is as we want.